.github/workflows/pr-ci.yml

```diff
@@ -21,10 +21,11 @@ jobs:
       branch: ${{ github.event_name == 'push' && github.ref_name || github.ref }}
 
       # When pushing to "main", publish and deploy to "dev" and "ops" (CD). For PRs, skip publishing and deploying (run CI only)
-      environment: ${{ (github.event_name == 'push' && github.ref_name == 'main') && 'ops' || 'none' }}
+      environment: ${{ (github.event_name == 'push' && github.ref_name == 'main') && 'dev,ops' || 'none' }}
 
       # Deploy provisioned plugin to Grafana Cloud
       grafana-cloud-deployment-type: provisioned
+      auto-merge-environments: dev,ops
```

Comment:
This can also be used in `publish.yml` in case you want to disable approvals for ops PRs when publishing a release to the catalog, it's up to you 🙂 

I decided to keep it as-is, meaning that Grafana Cloud deployments to ops will have to be approved manually after cutting a release

Reply:
@L2D2Grafana yes, prod will have to be approved manually in Argo, but dev and ops PRs originating for each commit from the main branch will be auto-merged.

With the current setup in `publish.yml`  (so for _manual releases_, not the CD-like ones off the main branch) however, ops deployments will have to be approved manually. If you want to require approvals only for prod deployments also for manual releases, you have to add `auto-merge-environments: dev,ops` also to `publish.yml` (the default value is: `auto-merge-environments: dev`).

Does this make sense?